src/main/java/com/dev/spring/controller/UserController.java

```diff
@@ -0,0 +1,65 @@
+package com.dev.spring.controller;
+
+import com.dev.spring.config.AppConfig;
+import com.dev.spring.dto.UserResponseDto;
+import com.dev.spring.model.User;
+import com.dev.spring.service.UserService;
+import java.util.ArrayList;
+import java.util.List;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.servlet.ModelAndView;
+
+@RestController
+public class UserController {
+    private static final Logger LOGGER =
+            LogManager.getLogger(UserController.class);
+
+    private final AnnotationConfigApplicationContext context =
+            new AnnotationConfigApplicationContext(AppConfig.class);
+    private final UserService userService = context.getBean(UserService.class);
+
+    @GetMapping(value = "/user")
+    public List<UserResponseDto> getAll() {
+        List<UserResponseDto> dtoList = new ArrayList<>();
+        userService.listUsers().forEach(user -> dtoList.add(new UserResponseDto(user)));
+        return dtoList;
+    }
+
+    @GetMapping(value = "/")
```

Comment:
```suggestion
    @GetMapping
```
should work this way too